### PR TITLE
Ballerina

### PR DESCRIPTION
**Pull request by agentelinux**
Add new lang to platform: 
* https://ballerina.io/

Thanks tsuru

### CODE REVIEW

**Comment by smiles-krishna** [comment]
@renatosis Bom dia.
Algo mais que posso fazer para ajudar?
[]s
Krishna

**Comment by smiles-krishna** [comment]
@morpheu você consegue ajudar a promover para master?
Obrigado
[]s
Krishna

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ sudo: required
 language: bash
 env:
   matrix:
-    - PLATFORM=buildpack
+    - PLATFORM=ballerina
+    - PLATFORM=buildpack
```
Comment (scorphus) [inline .travis.yml]: Can you please remove the trailing spaces from line 6?
```suggestion
    - PLATFORM=buildpack
```

File: nodejs/install [hunk 2]
```diff
@@ -7,7 +7,7 @@
 SOURCE_DIR=/var/lib/tsuru
 source ${SOURCE_DIR}/base/rc/config
 
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.7/install.sh | NVM_DIR=/etc/nvm bash
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | NVM_DIR=/etc/nvm bash
```
Comment (scorphus) [inline nodejs/install]: Can you please remove this change from this PR?

Comment (agentelinux) [reply nodejs/install]: the build of the nodejs failed then update the version.
Make sense?

Comment (agentelinux) [reply nodejs/install]: Error log
# ok 8 breaks with invalid node version

Comment (scorphus) [reply nodejs/install]: These are different things. If you believe that's an issue you can create a separate PR that has something else todo with other than Ballerina.

**Comment by agentelinux** [comment]
@scorphus is okay? tks

**Comment by scorphus** [comment]
Another suggestion I'd make, besides the new PR with the NVM upgrade (to a minor release, btw), is to squash all commits into a single one and get rid of all intermediate changes. But MHO here. It's up to you.

**Comment by agentelinux** [comment]
approve this PR and I open a new PR to NVM.
Okay?

**Comment by scorphus** [comment]
I'll definitely an happily aprove it if it doesn't include unrelated changes (i.e. NVM upgrade) 🙂

**Comment by agentelinux** [comment]
@scorphus , right. 
Done.

**Comment by agentelinux** [comment]
what's next step, to merge? :-)

**Pull request merged**